test/e2e/tls_test.go

```diff
@@ -55,20 +67,8 @@ func TestPeerTLS(t *testing.T) {
 		}
 	}()
 
-	members, err := e2eutil.WaitUntilSizeReached(t, f.KubeClient, 3, 60*time.Second, c)
+	_, err = e2eutil.WaitUntilSizeReached(t, f.KubeClient, 3, 60*time.Second, c)
```

Comment:
This is limited by SAN now and we don't have a good way to talk to k8s service.
I'm thinking about solutions. Any suggestions?
But before that I would like to add this test to guaranteed client TLS not breaking.